syntax = "proto3";

package PbData;

message RepeatedValue { repeated double values = 1; }

message ValuesMap { map<string, RepeatedValue> valuesMap = 1; }

message TimeValuesPack { map<string, ValuesMap> valuesPack = 1; }
